TGC/decisions/Germany VS Hungary Desc.txt

```diff
@@ -0,0 +1,63 @@
+political_decisions = {
+
+##Treaty of Vienna##
+
+treaty_of_Vienna_hungary = {
+		picture = treaty_signing
+		potential = {
+			
+			tag = GER
+                        has_global_flag = greatest_germany
+                        has_global_flag = hungarian_ultimatum_sent
+			ai = no
+			NOT = { has_global_flag = treaty_of_vienna_hungary }
+		}
+
+		allow = {			
+			controls = 641
+			controls = 652
+			controls = 633
+			controls = 624
+			controls = 657			
+		}
+
+		effect = {
+			
+			GER = { country_event = { id = 2000000000 days = 0 } }
+			
+		}
+		ai_will_do = { factor = 1 }
+}
+
+##Ultimatum to Hungary##

```

Comment:
Please have more descriptive descriptions (see USA.txt for examples).

Reply:
does this also apply for the events or are those fine?

Reply:
Generally everything should have at least a short description so that when somebody is looking for something they can most easily find it (without looking through the code). You can check out USA_Expansion.txt as an example for events.

Reply:
i thought them being in the seperate file should be enough to contextualise the descriptions i gave but i can clarify further if needed